### ISSUE TITLE
Patches fail to apply but pipeline suceeds

### ISSUE THREAD

**Issue by chris-durand** [modm-ext/freertos-partial#4]
The patches in the automatic update pipeline fail to apply:
```
error: while searching for:
    #define ipconfigNETWORK_MTU    1500
#endif

#ifndef ipconfigTCP_MSS
    #define ipconfigTCP_MSS    ( ipconfigNETWORK_MTU - ( ipSIZE_OF_IPv4_HEADER + ipSIZE_OF_TCP_HEADER ) )
#endif

error: patch failed: FreeRTOS-Plus-TCP/include/FreeRTOSIPConfigDefaults.h:395
error: FreeRTOS-Plus-TCP/include/FreeRTOSIPConfigDefaults.h: patch does not apply
```

The pipeline succeeds nevertheless. I'll fix the error reporting.

**Comment by salkinium** [comment]
I don't remember exacly, but I think I had to reorder these definitions to avoid undef errors from the defaults, maybe they fixed it now?

**Comment by chris-durand** [comment]
> I don't remember exactly, but I think I had to reorder these definitions to avoid undef errors from the defaults, maybe they fixed it now?

I guess so since our freertos worked fine without these patches. We'll see when I run it on the L552 tomorrow.

**Comment by salkinium** [comment]
Fixed in #5
